packages/mdc-textfield/mdc-text-field.scss

```diff
@@ -246,10 +249,6 @@
     // when the ripple is activated behind it.
     will-change: transform;
 
-    @include mdc-theme-dark(".mdc-text-field") {
```

Comment:
Why do these lines need to be removed? They don't appear to be related to the `:focused` state.